### PR TITLE
Updates to readme, pip info

### PR DESCRIPTION
**Pull request by swernli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Q# language and tooling
+# Modern QDK
```
Comment (billti) [inline README.md:1]: I think the term 'Modern' will be short lived, and fade into obscurity once the 'classic' QDK is forgotten about. I'd go with a heading of just 'Azure Quantum Development Kit'.

File: vscode/README.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
-# Azure Quantum Development Kit (QDK) Preview
+# Azure Quantum Development Kit (Modern QDK)
 
-**This extension is currently in preview. If you are looking for the current stable QDK, see the documentation
-at <https://learn.microsoft.com/en-us/azure/quantum/install-overview-qdk>**
+This extension brings rich Q# language support to VS Code. Develop, build, and run your Q# code from VS Code either locally on simulators, or by submitting a job to Azure Quantum.
 
-_Note: To install pre-release versions of this extension, you must click the drop-down
-next to the 'Install' button and select 'Install Pre-Release Version'_
+You can also try out this extension in VS Code for Web at [vscode.dev/quantum](http://vscode.dev/quantum).
```
Comment (billti) [inline vscode/README.md:5]: Should probably be an https link, not http :-)

File: vscode/README.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
 
-This extension brings rich Q# language support to VS Code. Develop, build, and run your Q# code from VS Code either locally on simulators, or by submitting a job to Azure Quantum.
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```
Comment (billti) [inline vscode/README.md:7]: Does this duplicate/conflict with the info on line 24 below

File: vscode/README-DEV.md [hunk 4]
```diff
@@ -5,3 +5,5 @@ of upcoming features.
 
 To install the tested version of this extension, visit <https://marketplace.visualstudio.com/items?itemName=quantum.qsharp-lang-vscode>.
 The README for the stable build contains details on features and usage.
+
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```
Comment (billti) [inline vscode/README-DEV.md:9]: I'd leave this readme minimal and referring to the stable extension for more info (as it does already).

**Review by billti — CHANGES_REQUESTED** [review]
Update line 3 of vscode/package.json to remove the word 'Preview' from the displayName also.

I'd limit use of the term 'modern' to where we're trying to disambiguate versions. I'm hoping this term is temporary until we the 'classic' QDK is a distant memory.

File: vscode/README.md (hunk 3 above)
Comment (swernli) [reply vscode/README.md:7]: Good catch!

**Comment by swernli** [comment]
> Update line 3 of vscode/package.json to remove the word 'Preview' from the displayName also.
> 
> 
> 
> I'd limit use of the term 'modern' to where we're trying to disambiguate versions. I'm hoping this term is temporary until we the 'classic' QDK is a distant memory.

Fixed the missing name change, and updated a url in that same file. 

I took out a few of the "Modern" usages, but we'll have to see what others think, since it seems there's some effort being put into making that moniker stick.

**Review by billti — APPROVED** [review]

File: pip/README.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ BellState()
 
 ## Support
 
-For more documentation and to browse issues, please visit the Q# project wiki at [https://github.com/microsoft/qsharp/wiki].
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```
Comment (minestarks) [inline pip/README.md:44]: Do we want the word "modern" here?

```suggestion
For more information about the Azure Quantum Development Kit, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```

**Review by minestarks — APPROVED** [review]

File: pip/README.md (hunk 5 above)
Comment (swernli) [reply pip/README.md:44]: a PM might say yes... but I don't feel too strongly. Saves us later updates if/when we want to reduce usage of "Modern"

**Pull request merged**